packages/docs/src/stories/components/expandable.stories.ts

```diff
@@ -9,7 +9,11 @@ const { generateTemplate } = storybookTemplate('sd-expandable');
 /**
  * Used to display a brief summary that expands to reveal additional content.
  *
- *  **Related templates**:
+ * **Accessibility Hints:**
+ * - The expandable component should only hide non-critical content, to ensure users can access essential information without unnecessary interaction.
+ * - Generic summaries like "show more" should be avoided.
```

Comment:
I think we should update our stories to reflect this recommendation, right now we are still showing "show more"